src/io.jl

```diff
@@ -0,0 +1,347 @@
+#-------------------------------------------------------------------------------
+# Ply file IO functionality
+
+#--------------------------------------------------
+# Header IO
+
+@enum Format Format_ascii Format_binary_little Format_binary_big
+
+function ply_type(type_name)
+    if     type_name == "char"   || type_name == "int8";    return Int8
+    elseif type_name == "short"  || type_name == "int16";   return Int16
+    elseif type_name == "int"    || type_name == "int32" ;  return Int32
+    elseif type_name == "int64";                            return Int64
+    elseif type_name == "uchar"  || type_name == "uint8";   return UInt8
+    elseif type_name == "ushort" || type_name == "uint16";  return UInt16
+    elseif type_name == "uint"   || type_name == "uint32";  return UInt32
+    elseif type_name == "uint64";                           return UInt64
+    elseif type_name == "float"  || type_name == "float32"; return Float32
+    elseif type_name == "double" || type_name == "float64"; return Float64
+    else
+        error("type_name $type_name unrecognized/unimplemented")
+    end
+end
+
+ply_type_name(::UInt8)    = "uchar"
```

Comment:
Can we please use the more descriptive (and Julia-friendly) names that PLY natively supports anyway as the default write names? (e.g. `uint8` instead of `uchar`.) It would be less confusing for humans reading the files.